client/src/components/inventory/HardwareList.tsx

```diff
@@ -1,14 +1,14 @@
 import React from "react";
-import HardwareItem from "./HardwareItem";
-import {Header, Icon, Item, Message} from "semantic-ui-react";
+import HardwareItem, {RequestedItem} from "./HardwareItem";
+import {Button, Header, Icon, Item, Message} from "semantic-ui-react";
 import PlaceholderItem from "./PlaceholderItem";
-import {RequestedItem} from "../inventory/HardwareItem";
-import {AppState} from "../../reducers/reducers";
 import {connect} from "react-redux";
-import {User} from "../../actions/actions";
-import Query from "react-apollo/Query";
-import {HwItem} from "../../types/ItemType";
+import {HwItem} from "../../types/Hardware";
```

Comment:
you already made the choice but I'm curious why you decided to extract all the types instead of keeping it together in the code area